transformer_engine/pytorch/csrc/common.h

```diff
@@ -145,7 +145,9 @@ class Float8CurrentScalingQuantizer : public Quantizer {
   at::Tensor scale_inv;
   at::Tensor amax;
   DType dtype;
+  bool with_computing_amax;
   bool with_amax_reduction;
+  bool with_computing_scale;
```

Comment:
If you need these options, it's a sign that you're doing something more advanced and CS-specific than the quantizer was designed for.
```suggestion
  bool with_amax_reduction;
```

Reply:
I think Kunlun needs these flags to skip amax calculation in the quantize function itself. The idea is that he could call the compute amax directly in his `cast_master_weights_to_fp8` for a list of master weights, then aggregate the amax reduction of them into one NCCL call. I cannot think of a better way to achieve it.